src/main/java/by/custom/utilcalculator/domain/constants/Command.java

```diff
@@ -1,70 +1,111 @@
 package by.custom.utilcalculator.domain.constants;
 
+import javax.annotation.Nullable;
+
 public enum Command {
+    //command families - not commands actually, just markers
+    AGE("age", null),
+    ENGINE_VOLUME("volume", null),
+    N1_N3_WEIGHT("n1_n3_weight", null),
```

Comment:
I think you can merge   N1_N3_WEIGHT and WEIGHT into one WEIGHT. But for now, lets leave as it's.

Reply:
It is better to do it now. It was just my mistake to miss this particular family. Fixed